### PR TITLE
added santa

### PR DESCRIPTION
**Pull request by ChaoticNeutralGnome**
Here comes Santa Claus, here comes Santa Claus, right down Santa Claus Lane

### CODE REVIEW

File: Games/types/Mafia/roles/cards/NaughtyOrNice.js [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+                        if (!this.actor.role.data.visitors?.length)
+                            {
+                                let role = this.target.getAppearance("investigate", true);
+                                let alignment = this.game.getRoleAlignment(role);
+                                let naughtyOrNice;
+                                switch (alignment){
+                                    case "Village":
+                                        naughtyOrNice = "nice";
+                                        break;
+                                    case "Mafia":
```
Comment (starryburst) [inline Games/types/Mafia/roles/cards/NaughtyOrNice.js:29]: there's no case for mafia

**Review by starryburst — DISMISSED** [review]

File: Games/types/Mafia/roles/cards/NaughtyOrNice.js (hunk 1 above)
Comment (ChaoticNeutralGnome) [reply Games/types/Mafia/roles/cards/NaughtyOrNice.js:29]: there is tho

**Review by starryburst — APPROVED** [review]

File: Games/types/Mafia/roles/cards/NaughtyOrNice.js [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+
+        this.meetings = {
+            "Check List": {
+                states: [ STATE_NIGHT ],
+                flags: [ FLAG_VOTING ],
+                action: {
+                    labels: [ LABEL_INVESTIGATE, LABEL_ALIGNMENT ],
+                    priority: PRIORITY_ALIGNMENT_LEARNER,
+                    run: function () {
+                        if (!this.actor.role.data.visitors?.length)
```
Comment (pikulet) [inline Games/types/Mafia/roles/cards/NaughtyOrNice.js:20]: a small point: a guard clause can help reduce the nesting here

if (this.actor.role.data.visitors?.length) { return }

File: data/roles.js [hunk 3]
```diff
@@ -388,6 +388,13 @@ const roleData = {
                 "That person will be resurrected.",
             ],
         },
+        "Santa": {
+            alignment: "Village",
+            description: [
+                "Gives out various items at night.",
+                "Can visit someone at night to know their alignment if not visited.",
```
Comment (pikulet) [inline data/roles.js:395]: the second sentence sounds awkward to me

**Review by pikulet — APPROVED** [review]

File: data/roles.js (hunk 3 above)
Comment (ChaoticNeutralGnome) [reply data/roles.js:395]: What would you change it to?

Comment (pikulet) [reply data/roles.js:395]: Gives a present to one player every night.
If not visited, can also learn if another player is naughty or nice.

**Review by pikulet — DISMISSED** [review]

**Review by starryburst — APPROVED** [review]

**Pull request merged**